### DISCUSSION TITLE
Projects are missing...

### DISCUSSION THREAD

**Discussion by guylepage3** [community/community#3767] [Mobile] [answered]
Hi. Loving the app but I and many other teams I know of, cannot use the apps due to the fact they use "GitHub Projects" and there is no way to view and use Projects. Please add this feature as soon as possible. We would love to use this app. Thank you.

**Comment by rafaelurben** [comment]
I think this would be an important feature, because I often have some notes I want to add to a project on the go and as far as I know, this isn't possible on the mobile website either 🤔

**Comment by nplasterer** [comment, marked as answer]
Hey thanks for your feedback! Feel free to upvote on this topic related to Projects to help boost awareness of this request! - #3023
Also appreciate if you can share more details with us in the thread above 👆 - how would you expect to manage projects on mobile? Do you expect to view the full kanban board on mobile? What are the key project features that you think we should cater on mobile?

**Comment by DrHouston** [comment]
From what I remember the older version of GitHub Mobile had a Kanban Board feature but now it does not
I also am unable to add a new issue on mobile now - what is going on? How do I fix this?
![GitHub Moble NO KANBAN](https://github.com/community/community/assets/39123135/e955b97a-1ee4-463e-8c20-f7235da4e8fe)
![GitHub Moble NO WAY TO ADD NEW ISSUE](https://github.com/community/community/assets/39123135/e9b73a83-49b5-4284-bcc7-5a014ca5d5b0)
